tests/unit/superset/test_superset_input_generation.py

```diff
@@ -0,0 +1,210 @@
+            "platform.apolo.us/preset": "cpu-large",
+        },
+        "preset_name": "cpu-large",
+        "resources": {
+            "limits": {"cpu": "1000.0m", "memory": "0M"},
+            "requests": {"cpu": "1000.0m", "memory": "0M"},
+        },
+        "tolerations": [
+            {
+                "effect": "NoSchedule",
+                "key": "platform.neuromation.io/job",
+                "operator": "Exists",
+            },
+            {
+                "effect": "NoExecute",
+                "key": "node.kubernetes.io/not-ready",
+                "operator": "Exists",
+                "tolerationSeconds": 300,
+            },
+            {
+                "effect": "NoExecute",
+                "key": "node.kubernetes.io/unreachable",
+                "operator": "Exists",
+                "tolerationSeconds": 300,
+            },
+        ],
+    }
+    assert helm_params["supersetWorker"] == {
+        "affinity": {
+            "nodeAffinity": {
+                "requiredDuringSchedulingIgnoredDuringExecution": {
+                    "nodeSelectorTerms": [
+                        {
+                            "matchExpressions": [
+                                {
+                                    "key": "platform.neuromation.io/nodepool",
+                                    "operator": "In",
+                                    "values": ["cpu_pool"],
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        },
+        "podLabels": {
+            "platform.apolo.us/component": "worker",
+            "platform.apolo.us/preset": "cpu-large",
+        },
+        "preset_name": "cpu-large",
+        "resources": {
+            "limits": {"cpu": "1000.0m", "memory": "0M"},
+            "requests": {"cpu": "1000.0m", "memory": "0M"},
+        },
+        "tolerations": [
+            {
+                "effect": "NoSchedule",
+                "key": "platform.neuromation.io/job",
+                "operator": "Exists",
+            },
+            {
+                "effect": "NoExecute",
+                "key": "node.kubernetes.io/not-ready",
+                "operator": "Exists",
+                "tolerationSeconds": 300,
+            },
+            {
+                "effect": "NoExecute",
+                "key": "node.kubernetes.io/unreachable",
+                "operator": "Exists",
+                "tolerationSeconds": 300,
+            },
+        ],
+    }
+    assert helm_params["init"] == {
+        "adminUser": {
+            "email": "admin@superset.com",
+            "firstname": "Superset",
+            "lastname": "Admin",
+            "password": "admin",
+            "username": "admin",
+        }
+    }
+    assert helm_params["extraSecretEnv"]["SUPERSET_SECRET_KEY"]
+
+
+@pytest.mark.asyncio
+async def test_superset_values_generation_with_own_postgres(
```

Comment:
let's ensure in this case the postgres pod also has preset, ownersip, etc labels & tolerations